string name
geometry_msgs/PoseWithCovariance pose
geometry_msgs/TwistWithCovariance twist
string[] tags
monodrive_msgs/WheelState[] wheels
monodrive_msgs/OOBB[] oobbs